CHANGELOG.md

```diff
@@ -65,11 +65,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#1337](https://github.com/regen-network/regen-ledger/pull/1337) The `AddCreditType` proposal handler has been removed.
 - [#1342](https://github.com/regen-network/regen-ledger/pull/1342) The `NewKeeper` method in `ecocredit/marketplace` requires an `authority` address.
 - [#1342](https://github.com/regen-network/regen-ledger/pull/1342) The `AllowedDenom` proposal handler has been removed.
-
+- [#1452](https://github.com/regen-network/regen-ledger/pull/1452) The `NewModule` method in `ecocredit/module` requires an `authority` address and store `key`.
```

Comment:
i think it might be better to describe what the breaking change was (position of args changed), i was confused at first thinking "i thought we already required those things" 🤔

Reply:
These are new since the last release: https://github.com/regen-network/regen-ledger/blob/v4.0.0/x/ecocredit/module/module.go#L48-L53 and not recorded anywhere also in the change log.

Reply:
Was not sure whether I should retroactively fix or just add a new entry and went with the latter.

Reply:
works for me 👍🏻

Reply:
I was being lazy. I updated the changelog. No entry for changing the order necessary though if these are new. Added the two pull requests for when they were added. :+1: